cmsimple/functions.php

```diff
@@ -900,22 +900,43 @@ function meta($n)
     }
 }
 
+/**
+ * Returns an IMG tag if an $img is provided and is valid, and a literal string otherwise.  
+ *
+ * @param string $img Optional image file name, to be searched in $pth['folder']['templateimages']
+ *
+ * @param string $title Optional title for the image, also used as alternative to image
+ */
+function img_or_text($img, $title) {
```

Comment:
The name of this function should better conform to other newer functions; something like `XH_imgOrText`.

Reply:
Done.